webapp/javascript/pages/tagExplorer/components/TagsSelector.module.scss

```diff
@@ -1,40 +1,45 @@
-.title {
-  display: inline-block;
-  text-transform: uppercase;
-  padding: 5px;
-  color: var(--ps-select-modal-title);
-  font-weight: 700;
-}
+.tagsSelector {
+  width: auto;
 
-.tags {
-  padding: 0;
-  list-style: none;
-  margin: 0;
-  text-align: left;
-  min-height: 150px;
-  max-height: 440px;
+  .title {
+    display: inline-block;
+    text-transform: uppercase;
+    padding: 5px;
+    color: var(--ps-select-modal-title);
+    font-weight: 700;
+  }
 
-  li {
-    &.selected {
-      background-color: var(--ps-blue-highlight);
-    }
+  .tags {
+    max-width: 500px;
+    padding: 0;
+    list-style: none;
+    margin: 0;
+    text-align: left;
+    min-height: 150px;
+    max-height: 440px;
 
-    input {
-      border: none;
-      padding: 10px 30px;
-      width: 100%;
-      text-align: left;
-      white-space: nowrap;
-      overflow: hidden;
-      text-overflow: ellipsis;
+    li {
```

Comment:
I know this code was already like this, but I am going to say it just for us to sync about these things.

I don't see the value in nesting selectors using sass here, since we use css modules anyway it's already pretty scoped.

When using nesting it's easier to make things confusing. So I would suggest for next to just make them more or less flat.

Reply:
copy 👍